core/src/main/scala/chisel3/Layer.scala

```diff
@@ -117,26 +117,48 @@ object layer {
   /** Create a new layer block.  This is hardware that will be enabled
     * collectively when the layer is enabled.
     *
+    * This function automatically creates parent layers as the layer of the
+    * current layerblock is an ancestor of the desired layer.  The ancestor may
+    * be the current layer which causes no layer block to be created.  (This is
+    * not a _proper_ ancestor requirement.)
+    *
     * @param layer the layer this block is associated with
     * @param thunk the Chisel code that goes into the layer block
     * @param sourceInfo a source locator
+    * @throws IllegalArgumentException if the layer of the currnet layerblock is
+    * not an ancestor of the desired layer
```

Comment:
ScalaDoc is complaining about not being able to find `IllegalArgumentException`, maybe `java.lang.IllegalArgumentException`?

Reply:
🧐 I'll try that, thanks!